src/menu.cc

```diff
@@ -7,7 +7,7 @@
 Menu::Menu() {
   auto accels=Config::get().menu.keys;
   for(auto &accel: accels) {
-#ifdef JUCI_UBUNTU_BUGGED_MENU
+#ifdef JUCI_UBUNTU
     size_t pos=0;
```

Comment:
Can you explain what this does? I tried to define JUCI_UBUNTU to see if the xml changed, but it didn't it seems, or is it a char-set-fix?